docs/guides/migrate-rest-myst.md

```diff
@@ -0,0 +1,144 @@
+
+To start using MyST in your existing Sphinx project,
+first {ref}``install the `myst-parser` Python package <myst-parser:sphinx/intro.md#install-the-myst-parser>``
+and then {ref}`enable it on your configuration <myst-parser:parse-with-sphinx>`:
+
+```{code-block} py
+:caption: conf.py
+:emphasize-lines: 4
+
+extensions = [
+    # Your existing extensions
+    ...,
+    "myst_parser",
+]
+```
+
+Your reStructuredText documents will keep rendering,
+and you will be able to add MyST documents with the `.md` extension
+that will be processed by MyST-Parser.
+
+As an example, _this guide_ is written in MyST
+while the rest of the Read the Docs documentation is written in reStructuredText.
+
+```{note}
+By default, MyST-Parser registers the `.md` suffix for MyST source files.
+If you want to use a different suffix, you can do so by changing your
+`source_suffix` configuration value in `conf.py`.
+```
+
+## Converting existing reStructuredText documentation to MyST
+
+To convert existing reST documents to MyST, you can use
+the `rst2myst` CLI script shipped by [rst-to-myst].
+The script supports converting the documents one by one,
+or scanning a series of directories to convert them in bulk.
+
+After {doc}``installing `rst-to-myst` <rst-to-myst:index>``,
+you can run the script as follows:
+
+```console
+$ rst2myst convert docs/source/index.rst  # Converts index.rst to index.md
+$ rst2myst convert docs/**/*.rst  # Convert every .rst file under the docs directory
+```
+
+This will create a `.md` MyST file for every `.rst` source file converted.
+
+[rst-to-myst]: https://rst-to-myst.readthedocs.io/
+
+### Advanced usage of `rst2myst`
+
+The `rst2myst` accepts several flags to modify its behavior.
+All of them have sensible defaults, so you don't have to specify them
+unless you want to.
+
+These are a few options you might find useful:
+
+{option}``-d, --dry-run <rst-to-myst:rst2myst-convert.--dry-run>``
+: Only verify that the script would work correctly,
+  without actually writing any files.
+
+{option}``-R, --replace-files <rst-to-myst:rst2myst-convert.--replace-files>``
+: Replace the `.rst` files by their `.md` equivalent,
+  rather than writing a new `.md` file next to the old `.rst` one.
+
+You can read the full list of options in
+{doc}``the `rst2myst` documentation <rst-to-myst:cli>``.
+
+## Enabling optional syntax
+
+Some reStructuredText syntax will require you to enable certain MyST plugins.
+For example, to write [reST definition lists], you need to add a
+`myst_enable_extensions` variable to your Sphinx configuration, as follows:
+
+```{code-block} py
+:caption: conf.py
+myst_enable_extensions = [
+    "deflist",
+]
+```
+
+You can learn more about {doc}`other MyST-Parser plugins <myst-parser:syntax/optional>`
+in their documentation.
+
+[reST definition lists]: https://docutils.sourceforge.io/docs/user/rst/quickref.html#definition-lists
+
+## reStructuredText syntax not yet supported by MyST
```

Comment:
```suggestion
## Write reStructuredText within MyST Markdown
```

I think this is probably closer to what the reader might be wondering to themselves

Reply:
Thanks! Changed it slightly locally 👍🏽